packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1831,25 +2024,42 @@ export default class Meeting extends StatelessWebexPlugin {
     }
 
     return MeetingUtil.validateOptions({sendAudio, localStream: stream})
-      .then(() => this.updateTransceiver({
-        type: 'audio',
-        sendTrack: options.sendAudio,
-        receiveTrack: options.receiveAudio,
-        track,
-        transceiver: audioTransceiver,
-        peerConnection: this.mediaProperties.peerConnection,
-        previousMediaDirection: {
-          sendTrack: this.mediaProperties.mediaDirection.sendAudio,
-          receiveTrack: this.mediaProperties.mediaDirection.receiveAudio
+      .then(() => MeetingUtil.updateTransceiver(
+        {
+          type: 'audio',
+          sendTrack: options.sendAudio,
+          receiveTrack: options.receiveAudio,
+          track,
+          transceiver: audioTransceiver,
+          peerConnection: this.mediaProperties.peerConnection,
+          previousMediaDirection: {
+            sendTrack: this.mediaProperties.mediaDirection.sendAudio,
+            receiveTrack: this.mediaProperties.mediaDirection.receiveAudio
+          }
+        },
+        {
+          mediaProperties: this.mediaProperties,
+          roapSeq: this.roapSeq,
+          meeting: this,
+          id: this.id
         }
-      }))
+      ))
       .then(() => {
         this.mediaProperties.setLocalAudioTrack(track);
         this.mediaProperties.mediaDirection.sendAudio = sendAudio;
         this.mediaProperties.mediaDirection.receiveAudio = receiveAudio;
       });
   }
 
+  /**
+   * @param {Object} options
+   * @param {boolean} options.sendVideo
+   * @param {boolean} options.receiveVideo
+   * @param {MediaStream} stream optional
```

Comment:
You can show a param is optional by adding brackets around it `@param {MediaStream} [stream]`
https://jsdoc.app/tags-param.html#optional-parameters-and-default-values

Reply:
![the more you know](https://media1.giphy.com/media/l0IsIsuNJly1evlIY/giphy.gif?cid=e1bb72ff5d260a62395a795a777318f6&rid=giphy.gif)